lib/demos/reference/motion_demo_container_transition.dart

```diff
@@ -0,0 +1,540 @@
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 16.0),
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionSmallPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                    const SizedBox(width: 8.0),
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionSmallPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                    const SizedBox(width: 8.0),
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionSmallPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 16.0),
+                ...List<Widget>.generate(10, (index) {
+                  return OpenContainer<bool>(
+                    transitionType: _transitionType,
+                    openBuilder: (context, openContainer) {
+                      return const _DetailsPage();
+                    },
+                    tappable: false,
+                    closedShape: const RoundedRectangleBorder(),
+                    closedElevation: 0.0,
+                    closedBuilder: (context, openContainer) {
+                      return ListTile(
+                        leading: Image.asset(
+                          'assets/demos/motion/avatar_logo.png',
+                          width: 40,
+                        ),
+                        onTap: openContainer,
+                        title: Text(
+                          GalleryLocalizations.of(context)
+                                  .demoMotionListTileTitle +
```

Comment:
the formatting looks off here, is this what dartfmt produced?

Reply:
Yeah dartfmt produced it.